src/chapters/protocol.xml

```diff
@@ -80,6 +80,22 @@
             it supports Ogg Vorbis audio, but no video, and can accept both PNG and JPEG images. It
             wants to connect to localhost at port 5900, and is leaving the three other parameters
             blank.</para>
+        <para>Version 1.1.0 of Guacamole adds a couple of other features to the handshake phase.  First,
```

Comment:
I'm not sure about listing new changes within the documentation as a set of layered changes. I feel that the documentation for version X really should be a reference for version X first, with other relevant information (protocol is different in older versions) being secondary.

What do you think of instead describing things strictly from the perspective of the way things are in this version? I think it would make more sense for the section to describe the protocol of the latest version with a note about the behavior of older versions, rather than a section mainly describing older versions with a note about the behavior of the latest.

Reply:
Yeah, that's a good point.  I'll re-word it to just describe the `1.1.0` behavior.

Reply:
Should be good, now.